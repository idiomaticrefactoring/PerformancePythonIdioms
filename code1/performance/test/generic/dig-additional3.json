[{"id":54105,"opcode":"QUERY","status":"NOERROR","flags":["qr","rd"],"query_num":1,"answer_num":0,"authority_num":10,"additional_num":5,"opt_pseudosection":{"edns":{"version":0,"flags":[],"udp":4096}},"question":{"name":"ultradns.com.","class":"IN","type":"NS"},"authority":[{"name":"ultradns.com.","class":"IN","type":"NS","ttl":172800,"data":"pdns196.ultradns.com."},{"name":"ultradns.com.","class":"IN","type":"NS","ttl":172800,"data":"pdns196.ultradns.net."},{"name":"ultradns.com.","class":"IN","type":"NS","ttl":172800,"data":"pdns196.ultradns.org."},{"name":"ultradns.com.","class":"IN","type":"NS","ttl":172800,"data":"pdns196.ultradns.info."},{"name":"ultradns.com.","class":"IN","type":"NS","ttl":172800,"data":"pdns196.ultradns.biz."},{"name":"ultradns.com.","class":"IN","type":"NS","ttl":172800,"data":"pdns196.ultradns.co.uk."},{"name":"ultradns.com.","class":"IN","type":"NS","ttl":172800,"data":"ari.alpha.aridns.net.au."},{"name":"ultradns.com.","class":"IN","type":"NS","ttl":172800,"data":"ari.beta.aridns.net.au."},{"name":"ultradns.com.","class":"IN","type":"NS","ttl":172800,"data":"ari.gamma.aridns.net.au."},{"name":"ultradns.com.","class":"IN","type":"NS","ttl":172800,"data":"ari.delta.aridns.net.au."}],"answer":[{"name":"www.cnn.com.","class":"IN","type":"CNAME","ttl":93,"data":"turner-tls.map.fastly.net."},{"name":"turner-tls.map.fastly.net.","class":"IN","type":"A","ttl":8,"data":"151.101.193.67"},{"name":"turner-tls.map.fastly.net.","class":"IN","type":"A","ttl":8,"data":"151.101.65.67"},{"name":"turner-tls.map.fastly.net.","class":"IN","type":"A","ttl":8,"data":"151.101.1.67"},{"name":"turner-tls.map.fastly.net.","class":"IN","type":"A","ttl":8,"data":"151.101.129.67"}],"additional":[{"name":"pdns196.ultradns.com.","class":"IN","type":"A","ttl":172800,"data":"156.154.64.196"},{"name":"pdns196.ultradns.com.","class":"IN","type":"AAAA","ttl":172800,"data":"2001:502:f3ff::e8"},{"name":"pdns196.ultradns.net.","class":"IN","type":"A","ttl":172800,"data":"156.154.65.196"},{"name":"pdns196.ultradns.net.","class":"IN","type":"AAAA","ttl":172800,"data":"2610:a1:1014::e8"}],"query_time":72,"server":"192.5.6.30#53(192.5.6.30)","when":"Mon Sep 03 19:25:24 EST 2018","rcvd":432,"when_epoch":1536027924,"when_epoch_utc":null}]
