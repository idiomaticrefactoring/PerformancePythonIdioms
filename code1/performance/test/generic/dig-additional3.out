; <<>> DiG 9.11.3-1ubuntu1.1-Ubuntu <<>> ultradns.com NS @a.gtld-servers.net
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 54105
;; flags: qr rd; QUERY: 1, ANSWER: 0, AUTHORITY: 10, ADDITIONAL: 5
;; WARNING: recursion requested but not available

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 4096
;; QUESTION SECTION:
;ultradns.com.          IN  NS

;; AUTHORITY SECTION:
ultradns.com.       172800  IN  NS  pdns196.ultradns.com.
ultradns.com.       172800  IN  NS  pdns196.ultradns.net.
ultradns.com.       172800  IN  NS  pdns196.ultradns.org.
ultradns.com.       172800  IN  NS  pdns196.ultradns.info.
ultradns.com.       172800  IN  NS  pdns196.ultradns.biz.
ultradns.com.       172800  IN  NS  pdns196.ultradns.co.uk.
ultradns.com.       172800  IN  NS  ari.alpha.aridns.net.au.
ultradns.com.       172800  IN  NS  ari.beta.aridns.net.au.
ultradns.com.       172800  IN  NS  ari.gamma.aridns.net.au.
ultradns.com.       172800  IN  NS  ari.delta.aridns.net.au.

;; ANSWER SECTION:
www.cnn.com.		93	IN	CNAME	turner-tls.map.fastly.net.
turner-tls.map.fastly.net. 8	IN	A	151.101.193.67
turner-tls.map.fastly.net. 8	IN	A	151.101.65.67
turner-tls.map.fastly.net. 8	IN	A	151.101.1.67
turner-tls.map.fastly.net. 8	IN	A	151.101.129.67

;; ADDITIONAL SECTION:
pdns196.ultradns.com.   172800  IN  A   156.154.64.196
pdns196.ultradns.com.   172800  IN  AAAA    2001:502:f3ff::e8
pdns196.ultradns.net.   172800  IN  A   156.154.65.196
pdns196.ultradns.net.   172800  IN  AAAA    2610:a1:1014::e8

;; Query time: 72 msec
;; SERVER: 192.5.6.30#53(192.5.6.30)
;; WHEN: Mon Sep 03 19:25:24 EST 2018
;; MSG SIZE  rcvd: 432
